packages/vscode-extension/package.json

```diff
@@ -344,10 +354,22 @@
         "title": "View environment",
         "enablement": "fx-extension.isMultiEnvEnabled"
       },
+      {
+        "command": "fx-extension.viewEnvironmentWithIcon",
+        "title": "View environment",
+        "icon": "$(file-symlink-directory)",
+        "enablement": "fx-extension.isMultiEnvEnabled"
+      },
       {
         "command": "fx-extension.activateEnvironment",
         "title": "Activate environment",
         "enablement": "fx-extension.isMultiEnvEnabled"
+      },
```

Comment:
1. Why keep both xxx and xxxWithIcon?
2. Hide the command  in command palette?

Reply:
For now there are both icon and right click to trigger this feature.